python/README.md

```diff
@@ -0,0 +1 @@
+# Python JIT Proxy SDK
```

Comment:
good to include some commands to quickstart env and run

simple overview of sniper vs shotgun in example good too